3rd-party-licenses.md

```diff
@@ -50,98 +52,174 @@ https://github.com/selectize/selectize.js/blob/master/LICENSE
 
-### eslint-plugin-standard
+### jquery.fix.clone
 License: MIT  
-https://github.com/standard/eslint-plugin-standard/blob/master/LICENSE
+https://github.com/spencertipping/jquery.fix.clone/blob/master/README
 
-### eslint-plugin-promise
-License: ISC  
-https://github.com/xjamundx/eslint-plugin-promise#license
+### jQuery MiniColors: A tiny color picker built on jQuery
+License: MIT  
+https://github.com/claviska/jquery-minicolors/blob/master/LICENSE.md
 
-### eslint-plugin-node
+### FastClick
 License: MIT  
-https://github.com/mysticatea/eslint-plugin-node/blob/master/LICENSE
+https://github.com/ftlabs/fastclick/blob/master/LICENSE
 
-### eslint-plugin-import
+### hoverIntent jQuery Plugin
 License: MIT  
-https://github.com/benmosher/eslint-plugin-import/blob/master/LICENSE
+https://github.com/briancherne/jquery-hoverIntent/blob/master/jquery.hoverIntent.js
 
-## Images and components used to run Shopsys Framework on Docker
-### Postgres
-License: PostgreSQL License  
-https://www.postgresql.org/about/licence/
+### nestedSortable jQuery Plugin
+License: MIT  
+https://github.com/ilikenwf/nestedSortable/blob/master/README.md
 
-### Nginx
-License: BSD-2-Clause  
-http://nginx.org/LICENSE
+### normalize.css
+License: MIT  
+https://github.com/necolas/normalize.css/blob/master/LICENSE.md
 
-### Redis
-License: BSD-3-Clause  
-https://redis.io/topics/license
+### jQuery UI Touch Punch 0.2.3
+License: MIT or GPL Version 2  
+https://github.com/furf/jquery-ui-touch-punch/blob/master/jquery.ui.touch-punch.js
 
-### Erikdubbelboer/phpredisadmin
-License: Creative Commons Attribution 3.0  
-https://github.com/erikdubbelboer/phpRedisAdmin
+### Modernizr
+License: MIT  
+https://github.com/Modernizr/Modernizr/blob/master/LICENSE
 
-### Selenium/standalone-chrome
-License: Apache License 2.0  
-https://github.com/SeleniumHQ/docker-selenium/blob/master/LICENSE.md
+### jquery.cookie
+License: MIT  
+https://github.com/carhartl/jquery-cookie/blob/master/MIT-LICENSE.txt
 
-### Adminer
-License: Apache License 2.0 or GPL 2  
-https://github.com/vrana/adminer/blob/master/readme.txt
+## Libraries dynamically referenced via Composer
+Run `composer license` in your `shopsys-framework-php-fpm` container of your project to get the latest licensing info about all dependencies. 
 
-### Docker.elastic.co/elasticsearch/elasticsearch-oss
-License: Apache License 2.0  
-https://github.com/elastic/elasticsearch/blob/66b5ed86f7adede8102cd4d979b9f4924e5bd837/LICENSE.txt
+## Libraries dynamically referenced via npm
+Run these commands in your `shopsys-framework-php-fpm` container of your project to get the latest licensing info about all packages.
 
-### Php
-License: The PHP License  
-https://hub.docker.com/_/php/
+```
+npm install --no-save license-checker
 
-## Libraries dynamically referenced via Composer
-Run `composer license` in your project to get the latest licensing info about all dependencies. 
\ No newline at end of file
+./node_modules/.bin/license-checker
```

Comment:
When used in monorepo, this only lists the `license-checker` and its dependencies, as it is not run in the `project-base` directory.

Reply:
Done